docs/src/de/SysSpec-arc42.md

```
@@ -166,15 +167,15 @@ Zur Beschreibung der Backend-API ist Swagger-UI eingebaut, die über das Gateway
 
 ### Resourcenzuteilung
-| Service                             | CPU-Kerne  | RAM       |
-|:------------------------------------|:-----------|:----------|
-| Backend                             | 0,5        | 2,25 GB   |
-| API-Gateway Datenportal (Frontend)  | 0,25       | 1 GB      |
-| API-Gateway Selfserviceportal       | 0,25       | 1 GB      |
-| API-Gateway Adminportal             | 0,25       | 1 GB      |
-| EAI                                 | 0,25       | 0,5 GB    |
-| GEO-EAI                             | 0,5        | 1 GB      |
-| Document-Storage                    | 0,5        | 512 MB    |
+| Service                            | CPU-Kerne | RAM     |
```

markdownlint-cli2 (0.18.1)

Tables should be surrounded by blank lines

(MD058, blanks-around-tables)